src/backend/sql/gnc-slots-sql.cpp

```diff
@@ -1100,6 +1100,10 @@ gnc_sql_init_slots_handler (void)
     static GncSqlObjectBackend be_data =
     {
         GNC_SQL_BACKEND_VERSION,
+// This was GNC_ID_ACCOUNT. If somethine blows up, change it back,
+// make the registry store a std::tuple<std::string,
+// GncSqlObjectBackendPtr>, and check the first string against types
+// in the functions that are called on each backend.
         GNC_ID_ACCOUNT,
```

Comment:
I don't remember now what I tried, but since the registry stores a tuple whatever it was must have blown up.